packages/logger/src/logger/Formatters.ts

```diff
@@ -39,9 +39,10 @@ export function handleRecursiveErrorArray(error: Error | any[]): string | any[]
 }
 
 // This formatter checks if the `BOT_IDENTIFIER` env variable is present. If it is, the name is appended to the message.
-export function botIdentifyFormatter(botIdentifier: string) {
+export function botIdentifyFormatter(botIdentifier: string, runIdentifier: string) {
```

Comment:
these args arent optional but for some reason we check for existence anyway

Reply:
Technically, this check would also cover empty string.

I'm also not super confident that the types are always followed by the calling code, since logger creation happens in some places where we use raw js.

I'm okay to remove the checks, though, and I suspect everything will continue to work.

Reply:
Done.

Reply:
i think its probably good enough to keep for empty string, didint think about that